test/topology/test_mv.py

```diff
@@ -34,14 +34,14 @@ async def test_mv_tombstone_gc_setting(manager):
     be here and not in the single-node cqlpy.
     """
     cql = manager.cql
-    async with new_test_keyspace(cql, ksdef) as keyspace:
-        async with new_test_table(cql, keyspace, "p int primary key, x int") as table:
+    async with new_test_keyspace(manager, ksdef) as keyspace:
```

Comment:
> Some tests restart server and reset the driver connection
> so we cannot use the original cql Session for
> dropping the created keyspace in the `finally` block.

Why is the old session not usable? Sounds like a driver bug.

Reply:
Could be, but it shouldn't hold up this PR

Reply:
Agreed.